articles/understanding-systemd/index.md

```diff
@@ -0,0 +1,291 @@
+Here are a few things to note before proceeding:
+1. This is an indepth resource on the basic concepts of systemd 
+2. It requires familiarity with basic Linux commands.
+3. An understanding of Linux processes, daemons and cgroups is also required.
+4. The reference section contains links to resources referenced in this article
+
+---
+### Introduction
+
+The official website describes systemd (spelled with a small 's') as a suite of tools that forms the basic building block for a Linux system. systemd handles the boot process for Linux systems. As an init implementation, it has a PID of 1 like other init systems e.g System V, Upstart.
+
+### What's init?
+Before we delve further, let's talk more about what exactly init is. An init program is usually the first to start and the last to stop.
+It initializes and terminates programs and services crucial to the operation of Linux systems.
+
+It's also a [daemon](https://man7.org/linux/man-pages/man7/daemon.7.html) process that runs during the entire lifetime of all programs in the system. 
+
+It's an ancestor process to all processes. It also adpots orphaned or zombie processes. That is, after a child loses its parent, the init process becomes its new parent.
```

Comment:
There is a typo 'adpots', or rather explain what it means otherwise.

Reply:
This was a typo. Will fix.

Reply:
I've been waiting for you to update this part...

Reply:
Ow! Sorry about that.